src/routes/content/deleteProject.js

```diff
@@ -26,7 +27,14 @@ const deleteProject = async (roomId) => {
       })
       await matrixClient.leave(space.room_id).catch(console.log)
     })
+    // if the room had a canonical alias we delete it and remove it from the room directory
+    if (config.medienhaus.createCanonicalAliasOnPublish) {
+      await matrixClient.setRoomDirectoryVisibility(roomId, 'private')
+        .catch(console.debug)
+      await matrixClient.deleteAlias('#' + space.rooms[0].name.replace(/\s+/g, '-').toLowerCase() + '-' + localStorage.getItem('mx_user_id').substring(1))
```

Comment:
-If- we ever decide to change the "algorithm" how we generate those aliases, the code wouldn't work for projects anymore, that were created with the "old syntax".

Instead maybe we can make use of the `unstableGetLocalAliases` function, and then just delete all aliases for the given room at once:
<img width="986" alt="image" src="https://user-images.githubusercontent.com/706419/174590542-e8524472-b873-4d5c-875c-9e53c4799b01.png">